config/routes.rb

```diff
@@ -166,5 +166,15 @@ def set_up_flipper
 
   get "pages/:name", to: "static#page"
   get "/register", to: "static#register"
+  resources :account_requests, only: [:new, :create] do
+    collection do
+      get 'confirmation'
+      get 'confirm'
```

Comment:
This should be `put`/`patch` since you're modifying the record:
```suggestion
      patch 'confirm'
```

Reply:
@armahillo hmmm I see your point. I see where it can be beneficial but I wanted to avoid confusing the token as an `id` which might use for routes such as `/resources/:resource_id/`. It is possible that we'll want to look up account_requests by their actual primary key one day and I wanted to reserve it for that case.